frontend/src/scenes/experiments/ExperimentResult.tsx

```diff
@@ -183,9 +183,13 @@ export function ExperimentResult(): JSX.Element {
                         <div className="no-experiment-results p-4">
                             {!experimentResultsLoading && (
                                 <div className="text-center">
-                                    <b>There are no results for this experiment yet.</b>
+                                    <div className="mb-4">
+                                        <b>There are no results for this experiment yet.</b>
+                                    </div>
+                                    {!!experimentResultCalculationError && (
+                                        <div className="text-sm mb-2">{experimentResultCalculationError}</div>
```

Comment:
Assuming you'll revisit this after backend is done?

Reply:
yes indeed, what we have here is the current state of things (just render the error message in the results)